tests/test_PsfexPsf.py

```diff
@@ -286,6 +286,48 @@ def testPsfexDeterminerTooFewStars(self):
         with self.assertRaisesRegex(RuntimeError, "Failed to determine"):
             psf, cellSet = self.psfDeterminer.determinePsf(self.exposure, psfCandidateListShort, metadata)
 
+    def testPsfexDeterminerNoStars(self):
+        """Test the (Psfex) psfDeterminer with no stars at all."""
+        self.setupDeterminer(self.exposure)
+        metadata = dafBase.PropertyList()
+        psfCandidateListEmpty = []
+
+        with self.assertRaises(
+            lsst.meas.extensions.psfex.psfexPsfDeterminer.PsfexNoStarsError, msg="No psf candidates supplied."
```

Comment:
`msg` here isn't doing what you think: that's the message that would be printed if the assert fails (nothing raises, or a different exception raises). What you want here and below is `assertRaisesRegex(PsfexNoStarsError, "No psf candidates supplied.")`. See here: https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex